dipy/workflows/io.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import division, print_function, absolute_import
+
+import os
+import numpy as np
+import logging
+from dipy.io.image import load_nifti
+
+from dipy.workflows.workflow import Workflow
+
+
+class SummarizeData(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'summarize'
+        
+    
+    def run(self, input_files, out_dir=''):
+
+        """ Workflow for creating a binary mask
+
+        Parameters
+        ----------
+        input_files : variable string
+           Nifti1, bvals and bvecs files.
+        out_dir : string, optional
+           Output directory (default input file directory)
+        """
+
+        #io_it = self.get_io_iterator()
+        #logging.basicConfig(format='',)
```

Comment:
You can remove these

Reply:
Actually do you know how to remove INFO from logging?